.evergreen/ssdlc-report.sh

```diff
@@ -40,7 +40,7 @@ declare -r EVERGREEN_BUILD_URL_PREFIX="https://spruce.mongodb.com/version"
 declare -r GIT_TAG="r${PRODUCT_VERSION}"
 GIT_COMMIT_HASH="$(git rev-list --ignore-missing -n 1 "${GIT_TAG}")"
 set +e
-    GIT_BRANCH_MASTER="$(git branch -a --contains "${GIT_TAG}" | grep 'master$')"
+    GIT_BRANCH_MASTER="$(git branch -a --contains "${GIT_TAG}" | grep 'main$')"
```

Comment:
For clarity should this be `GIT_BRANCH_DEFAULT`?

Reply:
Yes, I agree. Renamed.